modules/apps/archived/youtube-web/package.json

```diff
@@ -1,6 +1,5 @@
 {
 	"dependencies": {
-		"frontend-js-web": "*"
```

Comment:
@ling-alan-huang, can we add the folder `modules/apps/archived/` to the exclusions of this SF rule in `source-formatter.properties`? The modules inside this directory usually are deprecated and aren't part of `ant all`.